client/elife-theme/src/index.js

```diff
@@ -5,49 +5,49 @@ export default {
   /* Colors */
   colorBackground: '#fff',
   colorPrimary: '#0288d1',
-  colorSecondary: '#888',
+  colorSecondary: '#f7f7f7',
   colorFurniture: '#E0E0E0',
-  colorBorder: '#aaa',
+  colorBorder: '#E0E0E0',
   colorBackgroundHue:
-    '#f5f5f5' /* marginally darker shade of the app background so that it can be used to divide the interface when needed */,
+    '#f7f7f7' /* marginally darker shade of the app background so that it can be used to divide the interface when needed */,
   colorSuccess: '#629f43',
-  colorError: '#b50000',
+  colorError: '#cf0c4e',
+  colourWarning: '#e65100',
   colorText: '#212121',
-  colorTextReverse: '#fff',
+  colorTextSecondary: '#888'
   colorTextPlaceholder: '#bdbdbd',
+  colorTextReverse: '#fff',
 
   /* Text variables */
-  fontInterface: 'Georgia, serif',
-  fontHeading: '"Avenir Next Webfont", Arial, Helvetica, sans-serif',
-  fontReading: 'Georgia, serif',
-  fontWriting: '"Courier 10 Pitch", Courier, monospace',
+  fontInterface: '"Noto Sans", Arial, Helvetica, sans-serif',
+  fontHeading: '"Noto Sans SemiBold", Arial, Helvetica, sans-serif',
+  fontReading: '"Noto Serif", serif',
+  fontWriting: '"Noto Sans", Arial, Helvetica, sans-serif',
   fontSizeBase: '16px',
-  fontSizeBaseSmall: '13px',
-  fontSizeHeading1: '36px',
-  fontSizeHeading2: '26px',
-  fontSizeHeading3: '22px',
-  fontSizeHeading4: '20px',
-  fontSizeHeading5: '18px',
-  fontSizeHeading6: '16px',
-  fontLineHeight: '32px',
+  fontSizeBaseSmall: '14px',
+  fontSizeHeading1: '30px',
+  fontSizeHeading2: '22px',
+  fontSizeHeading3: '18px',
+  fontSizeHeading4: '16px',
+  fontSizeHeading5: '14px',
+  fontSizeHeading6: '12px',
+  fontLineHeight: '24px',
```

Comment:
Based on the Typography page in invision, it looks like we need a different line height for each different heading + body small, as well as this 'normal' one?

Reply:
Yes, I think we need to add line heights for each heading type. PubSweet doesn't have these line height variables, so if we use them in some components we should pay attention not to share them back.